go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
 	github.com/jaypipes/ghw v0.6.0
 	github.com/jaypipes/pcidb v0.5.0
+	github.com/k8snetworkplumbingwg/network-attachment-definition-client v1.1.1-0.20201110193115-68e49246082c
```

Comment:
I know this is still WIP, but just letting you know that I push a few PRs since this. Just added some constant strings to the types.go. But may be worth moving so all the repos are on the same version.

Reply:
Done